### DISCUSSION TITLE
disabling of auto configuration of the db-scheduler & manual configuration does not appear supported using property files

### DISCUSSION THREAD

**Discussion by datasmithadvtech** [kagkarlsson/db-scheduler#567] [Q&A]
I added following
# Db-scheduler configuration
db-scheduler:
  threads: 5
  polling-interval: 5s
  heartbeat-interval: 5m
  immediate-execution-enabled: false
  delay-startup-until-context-ready: true

which appear to have no effect.
# Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

## Expected Behavior
expected startup not to be immediate, but delayed for spring boot (latest) running on mac OS.

## Current Behavior

--------
# For bug reports
**YOU MAY DELETE THE `For bug reports` SECTION IF A NEW FEATURE REQUEST.**

## Steps to Reproduce the bug
1.
1.
1.

## Context
- DB-Scheduler Version :  starter 14.0.1
- Java Version : 21
- Spring Boot (check for Yes) : [ Yes] 3.3.5<
- Database and Version : mysql 9.1.0

**Comment by datasmithadvtech** [comment]
setting property in property file worked, but not yaml file

also, i need scheduler created after app starts - scheduler now starts after app starts, however, scheduler is created during app startup, which i need deferred until after app starts. 

can we have this supported?

**Reply by kagkarlsson** [reply]
Malformed yml?
